scripts/fix-links.js

```
@@ -0,0 +1,233 @@
+		if (indexPathRegex.test(updatedContent)) {
+			updatedContent = updatedContent.replace(indexPathRegex, "/core/");
+			hasChanges = true;
+			console.log(`    Fixed index/ -> core/`);
+		}
+
+		// Fix relative paths that reference index/ -> core/
+		const relativeIndexRegex = /(\[.*?\]\([^)]*?)index\//g;
+		if (relativeIndexRegex.test(updatedContent)) {
+			updatedContent = updatedContent.replace(relativeIndexRegex, (match, prefix) => {
+				console.log(`    Fixed relative index/: ${match}`);
+				return prefix + "core/";
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)